lib/processRunner.js

```diff
@@ -75,46 +46,24 @@ module.exports = function () {
       container.type = 'node'
     }
 
-    if (envFile && !fs.existsSync(envFile)) {
-      return cb('Error: ' + envFile + ' does not exist')
-    }
-
-    if (container.specific && container.specific.environment) {
-      envArgs = generateEnvironment(container.specific.environment)
-    }
-
-    if (envFile) {
-      var envFileData = dotenv.parse(fs.readFileSync(envFile))
-      envArgs += ' ' + generateEnvironment(envFileData || {})
-    }
-
-    if (isWin) {
-      toExec = envArgs + ' ' + cmd
-    } else {
-      toExec = envArgs + 'bash -c "exec ' + cmd + '"'
+    env = Object.create(process.env)
+    if (container.environment) {
+      env = _.merge(env, container.environment)
     }
```

Comment:
Fair point and it's used in a few places where it's not really needed - lets leave for now and I'll review and send some separate PRs removing lodash from the whole codebase